src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -253,45 +250,42 @@ impl CopyInterpreter {
             Some(max_files)
         };
 
-        let mut all_source_file_infos = if force {
+        let all_source_file_infos = if force {
```

Comment:
Don't understand here why we need check the force option.
As my understanding, we should get the max_files files from the list is ok no matter whether the force is true or not.

Reply:
if force = false, files is listed and then filtered by copied files

Reply:
I see, thanks.